pkg/awsutil/sts.go

```diff
@@ -25,8 +25,9 @@ const (
 	AwsConsoleURL                      = "https://console.aws.amazon.com/"
 	DefaultIssuer                      = "Red Hat SRE"
 
-	assumeRoleMaxRetries   = 3
-	assumeRoleRetryBackoff = 5 * time.Second
+	assumeRoleMaxRetries          = 3
+	assumeRoleRetryBackoff        = 5 * time.Second
+	assumeCustomerRoleSessionName = "RH-SRE"
```

Comment:
Not a blocker, I feel that `bp-cli` may use ISV's for connecting to the cluster in the near future.  In that case, this logic may not be accurate. 
Ideally, we should move some of the assumed role logic to `bp-api`  endpoint where API decide how to determine the session-name based on some other factors.

Reply:
We've updated the API to determine the role session name that should be used, but the actual AssumeRole logic needs to be done client side since we use individual SRE credentials for the isolated backplane flow.

https://gitlab.cee.redhat.com/service/backplane-api/-/merge_requests/329